### DISCUSSION TITLE
Taking photo by laptop and phone is useless, and so hard

### DISCUSSION THREAD

**Discussion by khalmatay** [community/community#137942] [GitHub Education]
### Hi everyone,

How I can take a photo of proof my academic status from a camera of laptop or phone??? Try please do it, I want to see it process. Why I can't upload (not taking) my proofs of pdf or jpg format. Taking the photo is unreal.....
![image](https://github.com/user-attachments/assets/aaa4ca4d-031e-4db4-9c76-23addb3e2067)
I only can paste and upload my proofs in good quality, but can't taking photos.....................
pls github helpers write me and I can send my proofs university and erasmus+ program

**Comment by alx-mp4** [comment]
Hi @khalmatay, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!

**Reply by khalmatay** [reply]
Hi, thank you, How can I apply this issue? I did something similar but I can't open the ticket, could you send the link pls.

**Reply by alx-mp4** [reply]
Here you go - https://support.github.com/request
